service/lib/dinstaller/storage/proposal.rb

```diff
@@ -43,6 +44,21 @@ def available_devices
         disk_analyzer.candidate_disks
       end
 
+      # Label that should be used to represent the given disk in the UI
+      #
+      # The label has the form: "NAME, SIZE, [USB], INSTALLED_SYSTEMS".
```

Comment:
well, to be honest this looks really unflexible that label is driven by backend. I kind of expect that backend gives information and it is up to UI to decide what and how it will be showed.

Reply:
I agree. On the other hand, I believe it would be enough for this initial prototype.

Reply:
We agreed to do it this way during the planning session. Otherwise we would have to expose a lot of details about the disks. And we wanted something quick without growing the API a lot with representation of quite complex objects.

But I can add a note to reconsider that later. It's true that at some points we will need to represent the disks with something that is closer to any of these:

- https://github.com/storaged-project/blivet/blob/3.5-devel/doc/dbus.rst
- http://storaged.org/doc/udisks2-api/latest/gdbus-org.freedesktop.UDisks2.Block.html

But so far we decided to go for a simple approach.